gcloud-java-core/src/main/java/com/google/gcloud/RetryParams.java

```diff
@@ -48,12 +48,16 @@
 
   private static final long serialVersionUID = -8492751576749007700L;
 
+  /**
+   * Note that App Engine Standard Environment front-end modules have a 60 second deadline for HTTP
+   * requests. For that reason, we set the default total retry period to less than 60 seconds.
+   */
+  public static final long DEFAULT_TOTAL_RETRY_PERIOD_MILLIS = 50_000L;
```

Comment:
I thought about it some more and decided to stick with 50 seconds because users may not even think to read the RetryParams javadoc, since we set it by default.  So even though it's a particular use case that may result in only slightly more errors for users, I decided to leave it as is.